packages/sku/lib/configure.js

```diff
@@ -39,39 +39,44 @@ module.exports = async () => {
   ];
 
   // TODO: Remove this migration before releasing sku v15.
-  if (await shouldMigrateOldEslintConfig()) {
-    console.log("'.eslintignore' file detected. Attempting migration...");
+  const { shouldMigrate, eslintIgnoreExists } =
+    await shouldMigrateOldEslintConfig();
 
-    const customIgnores = migrateEslintignore({
-      hasLanguagesConfig: Boolean(languages && languages.length > 0),
-      target: paths.relativeTarget,
-    });
+  if (shouldMigrate) {
+    console.log("'.eslintignore' file detected. Attempting migration...");
 
-    if (customIgnores.length > 0) {
-      try {
-        await addEslintIgnoreToSkuConfig({
-          skuConfigPath: paths.appSkuConfigPath,
-          eslintIgnore: customIgnores,
-        });
-        console.log(
-          "Successfully migrated '.eslintignore' file to 'eslintIgnore' property in sku config.",
-        );
-      } catch (e) {
-        console.log("Failed to automatically migrate '.eslintignore' file");
-        console.log('Error:', e.message, '\n');
+    if (eslintIgnoreExists) {
+      const customIgnores = migrateEslintignore({
+        hasLanguagesConfig: Boolean(languages && languages.length > 0),
+        target: paths.relativeTarget,
+      });
+
+      if (customIgnores.length > 0) {
+        try {
+          await addEslintIgnoreToSkuConfig({
+            skuConfigPath: paths.appSkuConfigPath,
+            eslintIgnore: customIgnores,
+          });
+          console.log(
+            "Successfully migrated '.eslintignore' file to 'eslintIgnore' property in sku config.",
+          );
+        } catch (e) {
+          console.log("Failed to automatically migrate '.eslintignore' file");
+          console.log('Error:', e.message, '\n');
+
+          console.log('Please manually add the following to your sku config:');
+          console.log(
+            chalk.green('eslintIgnore:'),
+            chalk.green(JSON.stringify(customIgnores, null, 2)),
+          );
+        }
 
-        console.log('Please manually add the following to your sku config:');
         console.log(
-          chalk.green('eslintIgnore:'),
-          chalk.green(JSON.stringify(customIgnores, null, 2)),
+          "Please note that this is a best-effort migration and you should manually review the 'eslintIgnore' property in your sku config.",
         );
+      } else {
+        console.log("No custom ignores found in '.eslintignore' file");
       }
-
-      console.log(
-        "Please note that this is a best-effort migration and you should manually review the 'eslintIgnore' property in your sku config.",
-      );
-    } else {
-      console.log("No custom ignores found in '.eslintignore' file");
     }
 
     console.log("Deleting '.eslintrc' and '.eslintignore' files...");
```

Comment:
**suggestion:** This message can be misleading

Being more explicit in what we are actually removing can be more clear. If we say we're removing your `.eslintignore` file and you have none you may not believe the next thing sku tells you ;)

Reply:
There is one right below this one too that mentions both files.

Reply:
Maybe we can cover all cases by logging "Deleting `.eslintrc` and `.eslintignore` files if they exist"?

Reply:
We probably don't even need the success message, if it throws an error we'll likely catch it elsewhere.